frontend/src/components/StandUp/StandUp.js

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import Loader from "../../globalComponents/Loader/Tombstone";
+import { useState, useEffect, useCallback } from "react";
+import {
+    useFetchEmployeeStandUp,
+    useUpdateEmployeeStandUp,
+} from "./standUpHooks";
+import { FormControl } from "baseui/form-control";
+import { Button } from "baseui/button";
+import { Textarea } from "baseui/textarea";
+import { SIZE } from "baseui/input";
+import { AiOutlineCheckCircle } from "react-icons/ai";
+import NotificationManager from "react-notifications/lib/NotificationManager";
+import standUpQuestions from "./StandUpQuestions";
+
+function StandUp() {
+    const [value, setValue] = useState(Array(standUpQuestions.length).fill(""));
```

Comment:
why do you need this state and why do you need useEffect hooks written here. lets discuss tomorrow